tests/baselines/reference/declarationEmitParameterProperty.js

```diff
@@ -19,6 +19,6 @@ exports.Foo = Foo;
 
 //// [declarationEmitParameterProperty.d.ts]
 export declare class Foo {
-    bar: string | undefined;
+    bar?: string | undefined;

```

Comment:
I don't think this is correct. `Foo` is _guaranteed_ to have an own property of `bar`. The optionality of the constructor parameter property shouldn't carry over to the property definition on the class.

Reply:
This matches our current typesystem behavior:

```ts
class Foo {
  constructor(public bar?: string) {}
}
const a: Foo = {}; // valid
```